api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1387,6 +1388,127 @@ public void getObject(String bucketName, String objectName, String fileName)
     }
   }
 
+  /**
+   * Copy a source object into a new object with the provided name in the provided bucket.
+   *
+   * </p>
+   * <b>Example:</b><br>
+   *
+   * <pre>
+   * {@code minioClient.copyObject("my-bucketname", "my-objectname", "/my-sourcebucketname/my-objectsourcename");}
+   * </pre>
+   *
+   * @param destBucketName
+   *          Bucket name where the object will be created.
```

Comment:
Can we add another line here saying `Bucket will be created if doesn't exist`? or it is implicit?